backend/database/migrations/2022_11_16_134040_create_questions_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('questions', function (Blueprint $table) {
             $table->uuid('id')->unique();
             $table->string("question_id");
+            $table->string("question");
```

Comment:
this is not accepted. please add a new column using the update migration format. If I may ask, why add question column when question_id is exists already?

Reply:
if you need questions, use eloquent relationship model to get the questions using question_id

Reply:
I am the one creating the question , where would i get the question from?
It is the actual question that is asked . No column to take the questions.

Reply:
please check out how to use eloquent orm model.. it will give you all the quetsions using question_id.
and pls use the migration update migration file to add any column to the DB is needed at any point..

Reply:
Yes i know , i was just explaining why i need to add the questions column . I will add it using the update migration file now

Reply:
yeah I get you.. I said you don't need the question column since you have question_id on the table again.. or what's the essence of the question_id?

Reply:
then the question_id is not needed...

Reply:
I dont know what the questions id is for , there is no question anywhere , this is the questions table to create the question to be asked , i will delete the questions id column , should i do that ?

Reply:
yes please..

Reply:
done